meerkat/datapanel.py

```diff
@@ -729,6 +734,85 @@ def merge(
             suffixes=suffixes,
             validate=validate,
         )
+    
+    def sort(
+        self,
+        by: Union[str, List[str]],
+        ascending: Union[bool, List[bool]] = True,
+        kind: str = "quicksort",
+    ) -> DataPanel:
+        """ 
+        Sort the DataPanel by the values in the specified columns. Similar to 
+        ``sort_values`` in pandas.
+
+        TODO(Hannah): Implement this method – should likely just wrap 
+        meerkat.ops.sort.sort. 
+
+        Args:
+            by (Union[str, List[str]]): The columns to sort by.
+            ascending (Union[bool, List[bool]]): Whether to sort in ascending or 
+                descending order. If a list, must be the same length as `by`.Defaults 
+                to True.
+            kind (str): The kind of sort to use. Defaults to 'quicksort'. Options 
+                include 'quicksort', 'mergesort', 'heapsort', 'stable'.
+
+        Return:
+            DataPanel: A sorted view of DataPanel.
+
+
+        """
+
+        panda_col_type = '<class \'meerkat.columns.pandas_column.PandasSeriesColumn\'>'
+        numpy_col_type = '<class \'meerkat.columns.numpy_column.NumpyArrayColumn\'>'
+        tensor_col_type = '<class \'meerkat.columns.tensor_column.TensorColumn\'>'
+
+        keys = []
+        
+
+        if len(by) > 1:  # Sort with multiple column
+            tensor_col_name = "none"
+            panda_col_name = "none"
+            for col in by:
+                
+                curr_col_type = str(type(self[col]))
+                print(curr_col_type)
+                # Convert all columns to numpy type
+                if curr_col_type == panda_col_type:
+                    self[col] = self[col].values
+                    panda_col_name = col
+
+                if curr_col_type == tensor_col_type:
+                    self[col] = self[col].numpy()
```

Comment:
We don't need to convert the column to NumPy in the original `dp` , we can just add it to a list of keys and then pass a reversed version of that list to lexsort